index.bs

```diff
@@ -282,8 +296,8 @@ An attribution trigger is a [=struct=] with the following items:
 : <dfn>aggregatable values</dfn>
 :: An [=ordered map=] whose [=map/key|keys=] are [=strings=] and whose
     [=map/value|values=] are non-negative 32-bit integers.
-: <dfn>aggregatable dedup key</dfn>
-:: Null or a non-negative 64-bit integer.
+: <dfn>aggregatable dedup keys</dfn>
+:: A [=set=] of [=aggregatable dedup key=].
```

Comment:
I think this comment applies to existing problems elsewhere in the spec, but what does it mean for this field to be a set? In other words, do the 3 fields of the `aggregatable dedup key` structure need to be unique for each entry in the set?

If the set aspect is irrelevant, let's make this a list for simplicity.

Reply:
Updated to use list, thanks.